app/services/ipam/dhcp.rb

```diff
@@ -2,7 +2,10 @@ module IPAM
   class Dhcp < Base
     delegate :dhcp, :dhcp_proxy, :to => :subnet
     def suggest_ip
-      return unless subnet.dhcp?
+      unless subnet.dhcp?
+        logger.warn "Unable to suggest IP: subnet #{subnet.name} don't have a DHCP proxy associated, a proxy is not in subnets organization or IPAM is not set to DHCP"
```

Comment:
don't -> doesn't
the subnets organization
I'd also consider writing organization/location

Reply:
Done